ext/gd/config.m4

```diff
@@ -30,8 +30,8 @@ fi
 PHP_ARG_WITH(xpm-dir, for the location of libXpm,
 [  --with-xpm-dir[=DIR]      GD: Set the path to libXpm install prefix], no, no)
 
-PHP_ARG_WITH(freetype-dir, for FreeType 2,
-[  --with-freetype-dir[=DIR] GD: Set the path to FreeType 2 install prefix], no, no)
+PHP_ARG_ENABLE(freetype, for FreeType 2,
+[  --enable-freetype       GD: Enable FreeType 2 support], no, no)
```

Comment:
No need to change the switch name. It's related to an external dependency, so it's definitely a `with` switch.

Thanks.